packages/discord-api/package.json

```diff
@@ -0,0 +1,32 @@
+{
+	"name": "discord-api",
+	"version": "0.0.2",
+	"author": "Ian Mitchell",
+	"description": "TODO",
+	"license": "MIT",
+	"exports": "./dist/src/index.js",
+	"main": "./dist/src/index.js",
+	"typings": "dist/src/index.d.ts",
+	"type": "module",
+	"scripts": {
+		"build": "tsc",
+		"clean": "rm -r dist/",
+		"typecheck": "tsc --noEmit",
+		"prettier": "prettier src/**/*.ts --check",
+		"lint": "eslint ./src/**/*.ts",
+		"test": "jest"
+	},
+	"files": [
+		"dist",
+		"src"
```

Comment:
Why would you pack src too?

Reply:
Makes reading source code easier. It isn't a large download, and I love being able to see the source code locally when I'm testing things.